Phases/Assessments/Phase_1.md

```diff
@@ -0,0 +1,37 @@
+# Phase 1 Assessment
+
+The phase 1 assessment will test your ability to write basic JavaScript programs for the Node.js platform as well as your fundamental SQL knowledge.
+
+You will _not_ be assessed on your HTML, CSS, or other web-specific skills.
+
+## Skills Assessed
+
+- Programming (Beginner Level)
+- JavaScript (Beginner Level)
+- Node.js (Beginner Level)
+- Testing (Beginner Level)
+- SQL (Beginner Level)
+
+Some examples of specific things you _may_ be required to do as part of the assessment (this is **not** an exhaustive list):
+
+- Manipulate Arrays, Strings, and Objects in JavaScript
+- Use conditionals and iteration to solve a problem
+- Pull values out of a JSON object
+- Write tests for your code with Mocha and Chai
+- Use synchronous functions from the Node.js standard library
+- Read and write to files from a Node.js program
+- Design a relational database schema
+- Write SQL to query a small PostgreSQL database (1-4 tables)
+
+### Secondary Skills
+
+The following skills are _important_ but will likely not be _explicitly_ assessed for. You will need some fluency in them in order to complete the assessment.
+
+- Code Editor (Beginner Level)
+  - example: use Atom or Sublime to edit and save code
+- Git & GitHub (Beginner Level)
```

Comment:
Git is super important. I would love to see it assessed in some way. Maybe just a quiz?